src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -60,32 +58,6 @@ protected boolean addPointtoArray(Point somePoint, List someArray) {
         }
     }
 
-    //          Get ship by ID function, assumes user only selects from drop down menu on front end
-    /*public Ship getShipByID(String id) {
```

Comment:
No this was in the original model where only one ship existed and so if you call this function it wouldn't the updated one with the new ships it would call this one.